# Generated for NaoQi method ALMemory.unsubscribeToMicroEvent
# Allows modules to unsubscribe from a piece of data, in order to stop getting notified when the data changes.

# Name of the data.
string name
# Name of the module.
string moduleName
---


